deploy/scripts/src/accounts/vc-storage.ts

```diff
@@ -140,3 +142,27 @@ function getToken(r: Response): string {
   if (token !== null && typeof token === 'string') return token
   fail('token not found')
 }
+
+interface vcPayload {
```

Comment:
```suggestion
interface VCPayload {
```
Fix the SonarCloud code smell

Reply:
Fixed now